sbin/prepareWorkspace.sh

```diff
@@ -295,18 +295,32 @@ checkingAndDownloadingAlsa() {
   if [[ -n "$FOUND_ALSA" ]]; then
     echo "Skipping ALSA download"
   else
+
     ALSA_BUILD_URL="https://ftp.osuosl.org/pub/blfs/conglomeration/alsa-lib/alsa-lib-${ALSA_LIB_VERSION}.tar.bz2"
     curl -o "alsa-lib.tar.bz2" "$ALSA_BUILD_URL"
     curl -o "alsa-lib.tar.bz2.sig" "https://www.alsa-project.org/files/pub/lib/alsa-lib-${ALSA_LIB_VERSION}.tar.bz2.sig"
-    # WORKSPACE in preference as Alpine fails gpg operation if PWD > 83 characters
-    export GNUPGHOME="${WORKSPACE:-$PWD}/.gpg-temp"
+
+    ## Add Special Rules For Alpine Linx As This Doesnt Work In docker
+
+    if echo "${BUILD_CONFIG[OS_FULL_VERSION]}" | grep -qi "alpine" ; then
+      # Use /tmp for alpine in preference to $HOME as Alpine fails gpg operation if PWD > 83 characters
+      # Alpine also cannot create ~/.gpg-temp within a docker context
+      export GNUPGHOME="/tmp/.gpg-temp.$$"
+    else
+      export GNUPGHOME="$HOME/.gpg-temp.$$"
```

Comment:
Perhaps leave as `${WORKSPACE:-$HOME}` or `${WORKSPACE:-$PWD}` which is what it was previously